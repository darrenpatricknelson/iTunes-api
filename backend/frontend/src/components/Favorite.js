export default function Favorite() {
  return (
    <i
      title="Remove favorite"
      className="fa-solid fa-heart heartLogo heartLogoTrue"
    />
  );
}
