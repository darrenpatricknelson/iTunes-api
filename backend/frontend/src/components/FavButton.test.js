import React from 'react';
import Favorite from '../components/Favorite.js';
import renderer from 'react-test-renderer';

// Testing the icon to be red if favorite
// using snapshot

test('renders correctly', () => {
  const tree = renderer.create(<Favorite />).toJSON();
  expect(tree).toMatchSnapshot();
});
