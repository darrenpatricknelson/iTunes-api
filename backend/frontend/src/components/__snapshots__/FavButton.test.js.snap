// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<i
  className="fa-solid fa-heart heartLogo heartLogoTrue"
  title="Remove favorite"
/>
`;
